SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -671,12 +671,12 @@ void CFVMFlowSolverBase<V, R>::ComputeVorticityAndStrainMag(const CConfig& confi
 
     StrainMag(iPoint) = sqrt(2.0*StrainMag(iPoint));
     AD::SetPreaccOut(StrainMag(iPoint));
+    AD::EndPreacc();
 
     /*--- Max is not differentiable, so we not register them for preacc. ---*/
     strainMax = max(strainMax, StrainMag(iPoint));
     omegaMax = max(omegaMax, GeometryToolbox::Norm(3, Vorticity));
 
-    AD::EndPreacc();
```

Comment:
Won't this increase the memory used by the tape? Since it's now inside the main recording, it gets pushed onto the tape no?

Reply:
So what happens is that e.g. `omegaMax` depends on `Vorticity` which is not registered as input. I think this is probably done by intention, at least in case I understand the comment above correctly. However the tape test/debug mode detects that we've cut a dependency (through e.g. `Vorticity[2]` carrying a non-zero tag that appears inside a preaccumulation region where only tags "1337" are allowed).
What we can do is to either leave it this way (increasing memory, yes), to register `Vorticity` as input (increasing memory, too), or to enforce the correct tag alongside the intentional(?) omission of the max-dependencies.

Reply:
I thought that excluding the max computations was the clean solution that we are looking for but it made a regression test fail. So whatever the remaining dependency is, there seems to be one. So I chose to fix the preaccumulation region by adding `Vorticity` as an input.